Conan.VisualStudio.Core/ConanRunner.cs

```diff
@@ -24,7 +24,17 @@ public Task<Process> Install(ConanProject project, ConanConfiguration configurat
             string ProcessArgument(string name, string value) => $"-s {name}={Escape(value)}";
 
             var arguments = string.Empty;
-            if (_conanSettings != null)
+
+            string profile = project.getProfile(configuration);
+            if (profile != null)
+            {
+                string generatorName = generator.ToString();
+                arguments = $"install {Escape(project.Path)} " +
+                            $"-g {generatorName} " +
+                            $"--install-folder {Escape(configuration.InstallPath)} " +
+                            $"--profile {Escape(profile)}";
```

Comment:
it ignores `--build` and `--update` arguments, which are currently set in global settings

Reply:
Done!